src/FramePointerValidatorServiceImpl/FramePointerValidatorServiceImpl.cpp

```diff
@@ -12,17 +12,16 @@
 
 #include "FramePointerValidator/FramePointerValidator.h"
 #include "ObjectUtils/ElfFile.h"
-#include "OrbitBase/Result.h"
 #include "code_block.pb.h"
 
-namespace orbit_service {
+namespace orbit_frame_pointer_validator_service_impl {
```

Comment:
maybe without _impl?

Reply:
Hmm, the previous change has `_impl` (`crash_service_impl`). I don't have an opinion either way, though. Would you like me to remove it from both?

Reply:
yep, makes sense to drop impl for both, sorry I missed it.

Reply:
Only the namespace, yes?

Reply:
and module name

Reply:
Done for FramePointerValidatorService.